app/v1/models/request/submitBsas/selfEmployment/SeAdditions.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.submitBsas.selfEmployment
+
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+import play.api.libs.functional.syntax._
+
+case class SeAdditions(costOfGoodsBoughtDisallowable: Option[BigDecimal],
```

Comment:
Do we need **_Se_** for the file names? they already separated via package

Reply:
Income and Expenses have the "se" prefix to align with FHLExpenses, and FHLIncome in the other endpoint, and I decided to put it on Additions as well just for consistency. They don't necessarily need it, but it does make it clearer which model you're importing and stops Inteliji from getting confused and importing the wrong model.